src/components/DropArea/DropArea.stories.js

```diff
@@ -0,0 +1,89 @@
+import { SbDropArea, SbDropAreaUpload } from './index'
+
+const DropTemplate = args => ({
+  components: { SbDropArea },
+  props: Object.keys(args),
+  template: `
+    <SbDropArea
+      v-bind="{
+        accept,
+        maxFileSize,
+        maxFile,
+      }" />`
+})
+
```

Comment:
Will these descriptions be reviewed by someone on the team later? I think there are some punctuation adjustments to be made.

Reply:
I think we need a review of all the descriptions, for better documentation in the future